requirements.txt

```diff
@@ -64,7 +64,7 @@ python-dateutil==2.9.0
 contextlib2==21.6.0
 
 django-bootstrap3==24.2
-requirements/src/sorl-3.3-py2.py3-none-any.whl
```

Comment:
We can use our new 3.4 version of sorl here which is compatible with django 4.2: https://github.com/ccnmtl/sorl/releases/tag/3.4